modules/porous_flow/examples/multiapp_fracture_flow/fracture_diffusion/tests

```diff
@@ -0,0 +1,26 @@
+[Tests]
+[no_multiapp]
+  type = CSVDiff
+  input = no_multiapp.i
+  csvdiff = no_multiapp_frac_T_frac_T_0002.csv
+  requirement = "PorousFlow shall provide examples of heat flow in matrix-fracture systems, including an example where the matrix is 2D and the fracture is 1D and the fracture mesh conforms with the matrix mesh, without using a MultiApp approach"
```

Comment:
same

Reply:
This also looks OK.  The first has "... fracture mesh conforms with the matrix mesh, without using ..." and the second has "... fracture mesh does not conform with the matrix mesh, using ..."